src/main/app/drafts/tasks/claimDetails.ts

```diff
@@ -3,6 +3,6 @@ import { DraftClaim } from 'app/drafts/models/draftClaim'
 export class ClaimDetails {
 
   static isCompleted (claim: DraftClaim): boolean {
-    return claim.reason.isCompleted()
+    return claim.reason.isCompleted() && claim.timeline.rows.length > 0
```

Comment:
how about this? it would match the existing code
`claim.timeline.isCompleted()`?

Reply:
I wanted to do it this way, but then unit tests are broken (when executed from IDE GUI):
![image](https://user-images.githubusercontent.com/5647337/36850990-11a83f04-1d60-11e8-862d-2af48bd5845c.png)


code added to Timeline class:
```

  isCompleted (): boolean {
    return this.rows.length > 0
  }
```

Reply:
OK, sorted. `deserialize` returns `MultiRowForm<TimelineRow>` to make it working well:

in `DraftClaim`:

```
this.timeline = new Timeline().deserialize(input.timeline) as Timeline
```